corehq/apps/style/templates/style/bootstrap3/base.html

```diff
@@ -228,7 +228,7 @@
     <body>
         {% maintenance_alert %}
         <!-- .hide used for A/B test of account creation page -->
-        <div class="hq-container{% if is_register_user_experiment %} hide{% endif %}">
+        <div class="hq-container{% if is_register_user %} molly{% endif %}">
```

Comment:
> is is_register_user ever false?

It's usually not defined - this template is used by most B3 pages, but only the reg user page passes in this flag.

> could potentially use a more descriptive (in general ways) class name

going to leave it alone for now, I think @biyeun is interested in making some changes to the "molly" version of this page anyway